src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetWorkTypesControllerCFTTest.java

```diff
@@ -0,0 +1,124 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.controllers;
+
+
+import io.restassured.http.Headers;
+import io.restassured.response.Response;
+import org.assertj.core.util.Lists;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import uk.gov.hmcts.reform.wataskmanagementapi.SpringBootFunctionalBaseTest;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.lessThanOrEqualTo;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.reform.wataskmanagementapi.services.SystemDateProvider.DATE_TIME_FORMAT;
+
+public class GetWorkTypesControllerCFTTest extends SpringBootFunctionalBaseTest {
+
+    private static final String ENDPOINT_BEING_TESTED = "work-types";
+
+    private Headers authenticationHeaders;
+
+    @Before
+    public void setUp() {
+        authenticationHeaders = authorizationHeadersProvider
+            .getTribunalCaseworkerAAuthorization("wa-ft-test-r2-");
+    }
+
+    @Test
+    public void should_return_work_types_when_user_has_work_types() {
+        common.setupOrganisationalRoleAssignmentWithWorkTypes(authenticationHeaders);
+
+        Response result = restApiActions.get(
+            ENDPOINT_BEING_TESTED + "/?filter-by-user=true",
+            authenticationHeaders
+        );
+        result.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .and().contentType(MediaType.APPLICATION_JSON_VALUE);
+
+        final List<Map<String, String>> workTypes = result.jsonPath().getList(
+            "work_types");
+
+        List<Map<String, String>> expectedWarnings = Lists.list(
```

Comment:
should this variable be expectedWorkTypes instead of expectedWarnings?

Reply:
you are correct, I will rename it.